pom.xml

```diff
@@ -142,7 +142,7 @@
     <dropwizard-metrics.version>4.1.2</dropwizard-metrics.version>
     <snappy-java.version>1.1.1.7</snappy-java.version>
     <log4j.version>2.11.2</log4j.version>
-    <netty.version>4.1.42.Final</netty.version>
+    <netty.version>4.1.54.Final</netty.version>
```

Comment:
Not sure whether it is possible that we stick to the current netty version and jackson version here, as any of these version bump could make big effect on the whole pinot cluster; it could bring in backward incompatibility between different pinot components during deployment.

Reply:
No, can't do that. I tried to and failed.
I would say that is what the travis checks are for.